File: Latihan Kelas/KendaraanInheritance/KoleksiDriver.cpp
#include "Kendaraan.cpp"
#include "Mobil.cpp"
#include "Minibus.cpp"
#include "Bus.cpp"
#include "KoleksiKendaraan.cpp"
#include <iostream>


int main(){
    KoleksiKendaraan* k = new KoleksiKendaraan();
    Mobil* m1 = new Mobil(1, 2000, "Toyota", "Owen");
    m1->printInfo();
    *k << *m1;
    k->printAll();

    // KoleksiKendaraan k;
    // k << Mobil(1, 2000, "Toyota", "Owen");
    // k << Bus(2, 2001, "Honda", 10);
    // k << Minibus(3, 2002, "Honda");
    // k.printAll();
}

File: Latihan Kelas/KendaraanInheritance - Jawwa/KoleksiDriver.java
public class KoleksiDriver {
    public static void main(String[] args){
        KoleksiKendaraan k = new KoleksiKendaraan();
        Mobil m1 = new Mobil(1, 2000, "Toyota", "Owen");
        Minibus mn1 = new Minibus(2, 2005, "Honda", "Amar");
        k.add(m1);
        k.add(mn1);
        k.printAll();
    }
}
